src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery first = new Lottery();
+        System.out.println(first.getRandomBall());
+        Lottery second = new Lottery();
+        System.out.println(second.getRandomBall());
+        Lottery third = new Lottery();
+        System.out.println(third.getRandomBall());
```

Comment:
why do you have to create three instances of lottery? just one is enough
let's also use for loop to create and print info about random balls